src/main/java/com/booksaw/betterTeams/commands/teama/AllyTeama.java

```diff
@@ -15,8 +15,8 @@ public class AllyTeama extends SubCommand {
 	@Override
 	public CommandResponse onCommand(CommandSender sender, String label, String[] args) {
 		
-		Team team1 = Team.getTeam(args[0]);
-		Team team2 = Team.getTeam(args[1]);
+		Team team1 = Team.getTeam(args.length >= 1 ? args[0] : null);
```

Comment:
We do not need to check arugments length as `getMinimumArguments()` leaves it to ParentCommand to handle

Reply:
yeah, IntelliJ doesn't knew that one (neither did I ;))